docs/database-structure.md

```diff
@@ -0,0 +1,336 @@
+# Schema Public
+
+## Tables
+
+### audit
+
+Some exercises require reviews from other students or any other member of the school. The reviewer checks that some conditions are met by the code of the student being reviewed. This process is called an `audit` and it's kept in this table.
+
+Columns:
+- id.
+- groupId: The group that the audit refers to.
+- auditorId: The user who's making the audit.
+- attrs: The attributes of the audits keep the feedback of the auditor for each question failed.
+- grade: 0 for failed and 1 for passed.
+- createdAt: The timestamp of the time and date of creation.
+- updatedAt: The timestamp of the last update.
+- code: The code that must be used by the auditor to start the audit.
```

Comment:
this code is set to null after audit is expired (important info to get the current audits)

Reply:
Really? this is so hacky

Reply:
Yeah, for sure it's  important to put it in the documentation. Thank you

Reply:
yes lol it's a little bit tricky if you don't know it
I thought it was logical, without code you cannot validate anymore, so it's not valid anymore 🤷🏼‍♀️ it was a way not to add a column with a written status, the shape of the data gives the status